cypress/e2e/urlHandling.cy.ts

```diff
@@ -30,6 +30,22 @@ describe('URL handling', () => {
 
       cy.title().should('eq', 'Test | Playroom');
     });
+
+    it('editor hidden', () => {
+      cy.visit(
+        'http://localhost:9000/#?code=N4IgpgJglgLg9gJwBJQhMA7EAuGCCuYAvkA'
+      );
+
+      cy.get('textarea').should('not.be.focused');
+
+      // Todo - write a test that checks the CodeMirror element is not visible
+      /*
+      The CodeMirror element is not visible, but it is in the DOM
+      This test fails because the element doesn't meet Cypress's requirements for being hidden
```

Comment:
Could be worth clarifying that we don't know what's causing it to fail the hidden requirement either.